.gitignore

```diff
@@ -14,3 +14,12 @@ tests/data/*.png
 # ignore doc products
 doc/*.md5
 doc/*.ly
+doc/*.svg
+doc/*.png
+doc/*.1
+
+# ignore the configure/make products
+autom4te.cache/
+config.*
+configure
+Makefile
```

Comment:
whats the weird character on the end of the line?

Reply:
that that's where the file ends (there's no newline after line 25)

Reply:
oh, its meant to be a zero followed by a  return indicating missing return at end of file, damn github :)

Reply:
I think it would be best to follow the convention of having the last newline as all other files do, see also dockerfile.

Reply:
Fixed. There's now a newline (though Github doesn't show them I guess?)

Reply:
Will take the absence of the "funny symbol" as proof :)